src/conversion/buildviewconverter.js

```diff
@@ -362,6 +363,86 @@ class ViewConverterBuilder {
 	}
 
 	/**
+	 * Registers how model element marking marker range will be created by converter.
+	 *
+	 * Created element has to match the following pattern:
+	 *
+	 * 		{ name: '$marker', attribute: { data-name: /^\w/ } }
+	 *
+	 * There are two ways of creating this element:
+	 *
+	 * 1. Makes sure that converted view element will have property `data-name` then converter will
+	 * automatically take this property value. In this case there is no need to provide creator function.
+	 * For the following view:
+	 *
+	 *		<marker data-name="search"></marker>foo<marker data-name="search"></marker>
+	 *
+	 * converter should look like this:
+	 *
+	 *		buildViewConverter().for( dispatcher ).fromElement( 'marker' ).toMarker();
+	 *
+	 * 2. Creates element by creator:
+	 *
+	 * For the following view:
+	 *
+	 * 		<span foo="search"></span>foo<span foo="search"></span>
+	 *
+	 * converter should look like this:
+	 *
+	 * 		buildViewConverter().for( dispatcher ).from( { name: 'span', { attribute: foo: /^\w/ } } ).toMarker( ( data ) => {
+	 * 			return new Element( '$marker', { 'data-name': data.getAttribute( 'foo' ) } );
+	 * 		} );
+	 *
+	 * @param {Function} [creator] Creator function.
+	 */
+	toMarker( creator ) {
+		const eventCallbackGen = function( from ) {
```

Comment:
I'm also for named function instead of anonymous function assigned to const, but this is consistent with the other methods. I'll fix it.